### PR TITLE
Documenting Documentation

### PR DESCRIPTION
**Pull request by avigoldman**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# Contributing
+
+Thank you for your interest in contributing to the SparkPost Developer Site!
+
+## API reference
+
+The API reference broadly follows [API Blueprint](https://apiblueprint.org/), an an open source extension of markdown designed for writing API specs, with a [couple of additions](#api-blueprint-extensions).
```
Comment (jasonrhodes) [inline CONTRIBUTING.md]: an an open source extension

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+# Contributing
+
+Thank you for your interest in contributing to the SparkPost Developer Site!
+
+## API reference
```
Comment (jasonrhodes) [inline CONTRIBUTING.md]: These headings _might_ work a little better as actions? Something like
* Updating the API Reference
* Editing the Table of Contents
etc?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+The table of contents is set in the `table-of-contents.json` file. It contains an array of "category" objects. Each of those objects has a `pages` key which contains an array of "page" objects.
+
+Each page object should have the following:
+
+* `title` - The title to be used in the navigation bar.
+* `file` - The source file.
+* `path` - The URL for the page.
+* `label` - (Optional) An additional label to be shown next to the navigation link. This is great for calling out a new endpoint.
+
+When adding a new API reference page, be sure to add it to the table of contents. If a file is not listed in the table of contents, it doesn't get generated.
```
Comment (jasonrhodes) [inline CONTRIBUTING.md]: This comment would be helpful in the "API Reference" section, too.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jasonrhodes) [reply CONTRIBUTING.md]: Hm I misunderstood the grouping here so my suggestion won't work as-is, but I think "verbing" those headings would help if you can figure out a way to do something like that... :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+Each API Blueprint file starts with metadata. At minimum, it needs the `FORMAT` and `title`.
+
+* `FORMAT` - Denotes the document is an API Blueprint file. It should always be `1A`.
+* `title`  - Use to set the page `<title>` tag.
+* `description` - Use to set the page description.
+* `full` - Use this to set the layout to be a single column. Set to `true` for pages that are purely markdown.
+
+### Content
+
+Describing how to write API files that talk about content.
```
Comment (jasonrhodes) [inline CONTRIBUTING.md]: ?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+The markdown directly following the title should include the Introduction and {Resource} Object sections.
+
+#### [Resources](https://apiblueprint.org/documentation/tutorial.html#resource)
+
+Resources tend to add noise and lead to poor organization (i.e. grouping create and list together simply because they have the same URI.)
+
+Avoid using them, unless you are creating a sub category.
+
+```md
+## Documentation [/documentation]
```
Comment (jasonrhodes) [inline CONTRIBUTING.md]: What is this code block here?

File: CONTRIBUTING.md (hunk 4 above)
Comment (avigoldman) [reply CONTRIBUTING.md]: I forgot to remove the incoherent note to myself. 😅

File: CONTRIBUTING.md (hunk 5 above)
Comment (avigoldman) [reply CONTRIBUTING.md]: It is an example "Resource"

**Pull request merged**